flarestack/data/icecube/ps_tracks/ps_v004_p00.py

```diff
@@ -30,7 +30,8 @@
 
 sample_name = "ps_tracks_v004_p00"
 
-logger.warning(
+# This dataset should be removed right away.
```

Comment:
Feel free to remove v004p00!

Reply:
Yes. I think there is really no use for it, but didn't want to mix it up with this PR. Will do shortly.